src/EditorFeatures/Test2/Simplification/CastSimplificationTests.vb

```diff
@@ -1568,7 +1740,7 @@ class Program
     {
         int x = int.MaxValue;
         double y = x;
-        double z = (float)x;
+        double z = x;
```

Comment:
this result is more desirable.  We can acutally remove this cast as the original location is a fixed-size location, not a FP location itself.